tests/parametric/test_telemetry.py

```diff
@@ -586,6 +595,63 @@ def test_telemetry_event_not_propagated(self, library_env, test_agent, test_libr
                 ), f"The install signature should not be included in the telemetry event, got {body}"
 
 
+@scenarios.parametric
+@features.telemetry_app_started_event
+class Test_TelemetrySSIConfigs:
+    """This telemetry provides insights into how a library was installed."""
+
+    @missing_feature(context.library == "python", reason="Not implemented")
+    @pytest.mark.parametrize(
+        "library_env",
+        [
+            {
+                **DEFAULT_ENVVARS,
+                "DD_INJECTION_ENABLED": "true",
```

Comment:
From digging through the [code](https://github.com/search?q=org%3ADataDog+DD_INJECTION_ENABLED&type=code&p=1) it looks like this should be a list of comma-separated values like `tracer`, `service_name,tracer` and similar

Can we update this to include test cases for `tracer`, `service_name,tracer`, and unset?